### ISSUE TITLE
Land on error page after clicking on the Tutorial dataset

### ISSUE THREAD

**Issue by nicOwlas** [labelflow/labelflow#487]
**Describe the bug**
Land on error page after clicking on the Tutorial dataset

**To Reproduce**
1. Load the tutorial dataset
2. See error page. Error in the console related to the query `workspaceSlugAndDatasetSlug`.
3. After a retry it worked

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/16015322/135815000-e128ad33-f8aa-4c07-a373-c8a7045e532f.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
```
LabelFlow
Debug





Debug information
Links:
Link to Github issue tracker
Link to Sentry error tracker
Link to Clarity usage analytics
Link to client bundle analysis
Link to server bundle analysis
Link to Graphiql
Link to Database admin
Link to CMS admin
Link to Github branch
Link to Github commit
Link to Vercel deployment
Link to CMS deployment on Heroku
Link to mailer provider
Client environment:
{
  "clientType": "Standard LabelFlow Client App",
  "isInWindowScope": true,
  "isInServiceWorkerScope": false,
  "isDevelopmentEnvironment": false,
  "browser": {
    "name": "chrome",
    "version": "94.0.4606",
    "os": "Mac OS",
    "type": "browser"
  },
  "env": {
    "NODE_ENV": "production",
    "NEXT_PUBLIC_VERCEL_ENV": "production",
    "NEXT_PUBLIC_VERCEL_URL": "labelflow-6080dya19-labelflow.vercel.app",
    "NEXT_PUBLIC_VERCEL_GIT_PROVIDER": "github",
    "NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG": "labelflow",
    "NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER": "labelflow",
    "NEXT_PUBLIC_VERCEL_GIT_REPO_ID": "366659969",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF": "main",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA": "49edbfb8bbf88cd940a45b7191dc951ff0eb0e09",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_MESSAGE": "Release 1.0.19\n\n[skip ci]",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_AUTHOR_LOGIN": "RaynalHugo",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_AUTHOR_NAME": "Hugo Raynal"
  }
}

Server environment:
{
  "serverType": "Standard LabelFlow Service Worker Server",
  "isInWindowScope": false,
  "isInServiceWorkerScope": true,
  "isDevelopmentEnvironment": false,
  "browser": {
    "name": "chrome",
    "version": "94.0.4606",
    "os": "Mac OS",
    "type": "browser"
  },
  "env": {
    "NODE_ENV": "production",
    "NEXT_PUBLIC_VERCEL_ENV": "production",
    "NEXT_PUBLIC_VERCEL_URL": "labelflow-6080dya19-labelflow.vercel.app",
    "NEXT_PUBLIC_VERCEL_GIT_PROVIDER": "github",
    "NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG": "labelflow",
    "NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER": "labelflow",
    "NEXT_PUBLIC_VERCEL_GIT_REPO_ID": "366659969",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF": "main",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA": "49edbfb8bbf88cd940a45b7191dc951ff0eb0e09",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_MESSAGE": "Release 1.0.19\n\n[skip ci]",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_AUTHOR_LOGIN": "RaynalHugo",
    "NEXT_PUBLIC_VERCEL_GIT_COMMIT_AUTHOR_NAME": "Hugo Raynal"
  }
}
```

**Comment by crubier** [comment]
Let’s merge hugo’s PR before working on this because it is very related

**Comment by crubier** [comment]
Fixed (as much as possible) in #486 

Basically we still have some possible bugs when there are new versions pushed to prod. But we tried to limit them.
